src/frontend/js/components/Spinner/_styles.scss

```diff
@@ -31,6 +31,11 @@
     border-top-color: r-theme-val(spinner, base-color-light);
   }
 
+  &--primary {
+    border-left-color: r-theme-val(spinner, base-color-primary);
+    border-top-color: r-theme-val(spinner, base-color-primary);
+  }
```

Comment:
The spinner is used in lots of places, we could do it but IMO we should also remove `--light` variant, and it will imply lots of changes in multiple files, we could do it in another PR?

Reply:
Indeed lets keep that